service/src/main/resources/db/changesets/20231208_clone-workflows-app-resources.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: Set cloneInstructions to COPY_RESOURCE for WORKFLOWS_APP resources.
```

Comment:
This is updating the CBAS database, not changing the WORKFLOWS_APP directly

Reply:
I guess I was thinking... "update resources associated with WORKFLOWS_APP". but you're right, it's more precise to describe them as being associated with the CBAS database